dirty_cat/datasets/datasets.py

```diff
@@ -0,0 +1,24 @@
+import pandas as pd
```

Comment:
Did you add pandas as a dependencies? In particular on the continuous integration?